sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -161,6 +161,16 @@ if (result.error !== undefined) {
 }
 ```
 
+This feature was introduced in TypeScript 3.2, so users of TypeScript 3.1 **must** cast result values to their corresponding success variant as follows:
```

Comment:
```suggestion
Strict null checks was introduced in TypeScript 3.2, so users of TypeScript 3.1 must cast result values to their corresponding success variant as follows:
```

Reply:
I think strict null checks were introduced in TS 2.x, but specifically 3.2 expanded the capability of discriminant tags.